python/pyspark/sql/column.py

```diff
@@ -251,15 +285,16 @@ def __iter__(self):
 
     # string methods
     _rlike_doc = """
-    Return a Boolean :class:`Column` based on a regex match.
+    SQL RLIKE expression (LIKE with Regex). Returns a boolean :class:`Column` based on a regex
```

Comment:
Let's leave so that it indicates the regular expression is in SQL syntax. I would like to keep them identically in most cases to reduce the overhead when someone needs to sweep the documentation.

It looks there are few places that needs the clarification. If this is something that has to be done, then, let's do this in another PR.